doc/source/reference/random/index.rst

```diff
@@ -78,7 +78,7 @@ bit generator-provided stream and transforms them into more useful
 distributions, e.g., simulated normal random values. This structure allows
 alternative bit generators to be used with little code duplication.
 
-The ``Generator`` is the user-facing object
+The `~.Generator` is the user-facing object
```

Comment:
Why not leave it as a link?

Reply:
Is ``Generator`` a link?  I though that was pre, and the `~.Generator` is a link?

Reply:
right, my mistake, reading the diff backwards